starters/graph-neural-networks/example_graph.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this need to be check if the image is inserted in the notebook or we need to copy separately...